packages/st2/debian/st2.st2auth.init

```diff
@@ -71,6 +71,8 @@ do_start()
   start-stop-daemon --start --quiet --chuid $RUNAS_USER:$RUNAS_GROUP --umask $UMASK --background --pidfile $PIDFILE \
      --exec $DAEMON -- $DAEMON_ARGS \
     || return 2
+  # wait for st2auth readiness to serve the requests
+  timeout 5 bash -c 'until echo > /dev/tcp/127.0.0.1/9100; do sleep 0.5; done' || return 2
```

Comment:
Looks like it's actually an issue with systemd (Ubuntu 16.04).

Auth service never starts and looks like systemd listens on 9100 and never releases that port and that's why service fails to start:

```bash
ubuntu@st2-pkg-unstable-u16-ent-dev-9e465edbf5:~$ sudo lsof -i :9100
COMMAND PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
systemd   1 root   47u  IPv4 181597      0t0  TCP localhost:9100 (LISTEN)
```